### PR TITLE
add arm64 support

### PR DESCRIPTION
**Pull request by briandowns**

### CODE REVIEW

**Review by brandond — APPROVED** [review]

**Comment by rbrtbnfgl** [comment]
Are you sure that it's building? If you change the building image version the `go-assert-boring.sh` script will fail for amd64. I fixed it there #20

**Review by rbrtbnfgl — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1,50 +1,47 @@
...
-             bin/host-local \
-             bin/ipvlan \
-             bin/macvlan \
-             bin/portmap \
-             bin/ptp \
-             bin/vlan ; \
-           fi \
-    && mkdir -vp /opt/cni/bin \
-    && install -D -s bin/* /opt/cni/bin
+RUN go-assert-static.sh bin/* && \
```
Comment (rbrtbnfgl) [inline Dockerfile:29]: I think there is an issue with the build on this line.
If you check the drone build it prints an error.
```
#10 0.347 sh: missing ]
#10 0.347 /bin/sh: amd64: not found
```

File: Dockerfile [hunk 2]
```diff
@@ -1,50 +1,47 @@
...
+    if [ "${ARCH}" != "s390x" || "${ARCH}" != "arm64" ]; then \
```
Comment (rbrtbnfgl) [inline Dockerfile:30]: I think it should be an and here not or.

Comment (rbrtbnfgl) [inline Dockerfile:30]:
```suggestion
    if [ "${ARCH}" != "s390x" ] && [ "${ARCH}" != "arm64" ]; then \
```

**Review by rbrtbnfgl — CHANGES_REQUESTED** [review]
With the changes on that if the build will fail at the `go-assert-boring.sh` that's why the `GOEPERIMENT` env is needed.

**Review by rbrtbnfgl — APPROVED** [review]

**Review by brandond — APPROVED** [review]

**Pull request merged**